### PR TITLE
ft: add reviews card component

### PR DESCRIPTION
**Pull request by SergeInGithub**
# Description

This will add the following component:
- Reviews card

The mentioned card will be show when a user is viewing a specific product and will then see reviews and feedback left on that particular product

📹   Loom video link: https://www.loom.com/share/df79a9aae292453cbf48f5143fb5ec7c

Fixes # (#39)

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shyakadavis — COMMENTED** [review]
Neat.

But some points.

1. The stars seem to be an image, but once a P.R with react icons is merged, you could opt to replace that with icons, and then conditionally render stars with a calculated logic.
2. The card is not responsive. And with a hard coded width, I don't think it is going to play well once it is actually being rendered within different containers/layouts.
3. Could you try to reduce the font size of the name, and rather increase the size of the avatar a bit. I believe it will contribute to the personality of the review a bit.

File: src/components/ui/ReviewsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}: ReviewsProps) {
+  return (
+    <div className="bg-white md:w-[700px] md:h-60 rounded-lg md:mt-2 drop-shadow-lg w-[300px] h-auto">
+      <div className="flex flex-row ml-5 pt-4">
+        <img className="w-11" src={image} alt="Profile" />
+        <h3 className="text-dark text-base font-semibold ml-5 pt-2">{name}</h3>
+      </div>
+      <div className="flex flex-row items-center ml-5 mt-3">
+        {/* Add stars component here */}
+        <img className="w-20" src={stars} alt="Ratings" />
```
Comment (mucyoalexaxel) [inline src/components/ui/ReviewsCard.tsx:23]:
```suggestion
```
use star icons instead of a picture. might be a bit challenging but will teach you a lot.

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]
@SergeInGithub Great work here. Just check a few suggested changes below, the rest looks good.

File: src/components/ui/ReviewsCard.tsx (hunk 1 above)
Comment (emilenfc) [reply src/components/ui/ReviewsCard.tsx:23]: Yes, it's pretty nice

**Review by emilenfc — DISMISSED** [review]
This look nice, change requested changes for for using star icons. Rest look good

File: .gitignore [hunk 2]
```diff
@@ -10,6 +10,7 @@ lerna-debug.log*
 build/
 
 node_modules
+coverage
```
Comment (shyakadavis) [inline .gitignore:13]: Not necessary. Check on the other P.Rs. I left a similar comment.

File: src/components/shared/ReviewsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import tyson from '../../assets/images/tyson.svg';
+import Stars from './Stars';
+
+type ReviewsProps = {
+  image?: string;
+  name?: string;
+  rating: number;
```
Comment (emilenfc) [inline src/components/shared/ReviewsCard.tsx:8]: Hello @SergeInGithub, I don't think it is good to display rating by Stars and again display number of rating

Comment (SergeInGithub) [reply src/components/shared/ReviewsCard.tsx:8]: The inspiration for this was got from the figma design, and we'll also later on use the rating to display the average number ratings @emilenfc

File: .gitignore (hunk 2 above)
Comment (mucyoalexaxel) [reply .gitignore:13]: @shyakadavis @SergeInGithub If there is a PR which was merged having this fix, a rebase will solve this and it won't be tracked as a new change.

Comment (shyakadavis) [reply .gitignore:13]: Indeed.

I was pointing out that he didn't need to include it once again.

Comment (SergeInGithub) [reply .gitignore]: @shyakadavis @mucyoalexaxel Thanks for noting this out, for some reason i had missed it, but it's all up fixed now.

**Review by shyakadavis — DISMISSED** [review]
LGTM

File: src/components/shared/ReviewsCard.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import tyson from '../../assets/images/tyson.svg';
```
Comment (musllim) [inline src/components/shared/ReviewsCard.tsx:2]: use avatar icon not this picture

File: src/components/shared/ReviewsCard.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  );
+}
+
+export default ReviewsCard;
+
+ReviewsCard.defaultProps = {
+  image: tyson,
+  name: 'Mike Tyson',
+  // rating: 3,
+  date: 'October 13, 2023',
```
Comment (musllim) [inline src/components/shared/ReviewsCard.tsx:43]: I think date should be required

**Review by musllim — COMMENTED** [review]
look at the reviews I left on default props and user avatar

**Review by shyakadavis — APPROVED** [review]
LGTM

**Review by musllim — APPROVED** [review]
LGTM

**Review by ISHIMWE-Aime — DISMISSED** [review]
LGM

**Review by emilenfc — APPROVED** [review]
LGTM

**Review by emilenfc — DISMISSED** [review]

**Review by emilenfc — APPROVED** [review]

**Review by shyakadavis — APPROVED** [review]
LGTM

**Review by ISHIMWE-Aime — APPROVED** [review]
Nice work 
LGTM

**Pull request merged**